helpers.lua

```diff
@@ -33,6 +33,10 @@ this.max_num = function(table)
     end
     return max
 end
+this.max_n = function(note_ids, n)
+    table.sort(note_ids)
+    return {table.unpack(note_ids, math.max(#note_ids - n + 1, 1), #note_ids)}
+end
```

Comment:
This name is hard to understand. Apparently, this function should be named something like `take_last_n_added_notes`.